src/client.js

```diff
@@ -1202,6 +1202,26 @@ export class StreamChat {
 		return this.get(this.baseURL + `/channeltypes`);
 	}
 
+	createCommand(data) {
```

Comment:
We need to update typescript [file](https://github.com/GetStream/stream-chat-js/blob/master/types/stream-chat/index.d.ts) for these changes. If you familiar with typescript, please do it ... or let me know. I will update it once this PR is approved.